cmd/upload.go

```diff
@@ -27,9 +27,11 @@ import (
 
 var (
 	uploadCmd = &cobra.Command{
-		Use:   "upload [LOCATION] [FILE]",
-		Short: fmt.Sprintf("Upload a file to %s", dist.CompanyName),
-		Args:  cobra.ExactArgs(2),
+		Use:     "report upload LOCATION FILE",
+		Aliases: []string{"upload"},
```

Comment:
```suggestion
		Use:     "upload LOCATION FILE",
```
No longer needed since this is under the report command and is hidden